src/ansys/fluent/core/utils/fluent_version.py

```diff
@@ -128,3 +128,9 @@ def __lt__(self, other):
     def __repr__(self) -> str:
         """Return a string representation for the Fluent version."""
         return self.value
+
+    def __str__(self) -> str:
+        """String output for the Fluent version."""
+        return (
+            f"Fluent version 20{self.value.split('.')[0]} R{self.value.split('.')[1]}"
+        )
```

Comment:
Please add a test for this, which helps to document the behaviour.

Reply:
Done, Thank you.